src/schema/opportunity.ts

```diff
@@ -576,6 +578,21 @@ export const typeDefs = /* GraphQL */ `
 
       state: ProtoEnumValue!
     ): EmptyResponse @auth
+
+    """
+    Create a shared Slack channel and invite a user by email
+    """
+    createSharedSlackChannel(
+      """
+      Email address of the user to invite
+      """
+      email: String!
+
+      """
+      Name of the channel to create (lowercase letters, numbers, hyphens, and underscores only)
+      """
+      channelName: String!
+    ): EmptyResponse @auth
```

Comment:
Should we check that user is recruiter/paid, like this any auth user can create slack channel?